pages/installation/os/debian.rst

```diff
@@ -13,7 +13,7 @@ Prerequisites
 Not all required dependencies are available in the standard repository, so we need to add `Debian Backports <https://backports.debian.org>`__ to the list of package sources::
 
   $ sudo echo "deb http://ftp.debian.org/debian jessie-backports main" > /etc/apt/sources.list.d/backports.list
-  $ sudo apt-get update 
+  $ sudo apt-get update && apt-get upgrade
```

Comment:
Either remove `sudo` from the first command or add it to the second command, but as it is, it's inconsistent.